compiler/src/dmd/backend/arm/cod1.d

```diff
@@ -292,7 +292,7 @@ int isscaledindex(tym_t ty, elem* e)
 @trusted
 void logexp(ref CodeBuilder cdb, elem* e, uint jcond, FL fltarg, code* targ)
 {
-    //printf("logexp(e = %p, jcond = %d)\n", e, jcond); elem_print(e);
```

Comment:
is the change related?

Reply:
I used it to track down the problem. I tend to leave them in, commented out, for future use.

Reply:
OK